### ISSUE TITLE
After pagination not showing record

### ISSUE THREAD

**Issue by nonuindu** [flaviusmatis/simplePagination.js#54]

**Comment by mmsheeks** [comment]
I think I might be having a similar problem to nonuindu. I have a table that I'm attempting to use this plugin with.

jQuery:
```javascript
$(function() {
    $('#tv-incompatibilities').pagination({
        items: 1200,
        itemsOnPage: 15,
        cssStyle: 'light-theme'
    });
)};
````

HTML
````html
<table id="tv-incompatibilities">
    <tr>...</tr>
    ...
</table>
````

Table works fine before applying the jQuery, but once the pagination is loaded, I only get the pager tool, not the table I expected. Maybe I'm misunderstanding how this will work?

**Comment by bilalakil** [comment]
Unfortunately you are misunderstanding how it works. The plugin basically creates the "pager tool" you've been seeing and provides an easy means to connect that to your SEPARATE table.

Your HTML would be something like this:

    <table id="tv-incompatibilities">
        ....
    </table>
    <div id="tv-pagination"></div>

Use the plugin to turn `#tv-pagination` into your pager.

Note that this won't magically make your table paginate either - it'll simply provide the tool and a means to easily "connect it" to your table.

The documentation has details on the events that fire when a user selects a page on the pager. You should use that event and the argument it provides to control your table as you wish.

If you need help with that, I've written a guide here: http://bilalakil.me/simplePagination

**Comment by mmsheeks** [comment]
Yep, Put two and two together shortly after posting. I appreciate the detailed response though. Thanks for the tool!

**Comment by mukeshjajoria** [comment]
after loading the page dynamically not able to show the page.only work for a static page. please suggest code.
